### ISSUE TITLE
waitFor is deprecated 

### ISSUE THREAD

**Issue by FraH90** [snobu/destreamer#337]
Today I had this puppetter error:

waitFor is deprecated and will be removed in a future release. See https://github.com/puppeteer/puppeteer/issues/6214 for details and how to migrate your code

What is this about?

**Comment by lukaarma** [comment]
Yeah I already knew, just it wasn't worth it to make a push only for this as I'm working on an update and I'll bundle this.
Don't worry its just a method becoming deprecated, it will be fixed in the next version of Destreamer
